examples/efs/filesystem_encrypted.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: efs.aws.upbound.io/v1beta1
+kind: FileSystem
+metadata:
```

```suggestion
metadata:
  annotations:
    meta.upbound.io/example-id: efs/v1beta1/filesystem
```
Could you please add the example id?